src/Controllers/Scheduler/shows.php

```diff
@@ -11,7 +11,8 @@
 use \MyRadio\ServiceAPI\ServiceAPI;
 use \MyRadio\ServiceAPI\MyRadio_Show;
 
-$shows = MyRadio_Show::getAllShows();
+// Get public shows (type 1) and check if all=true, in which case all shows, not just this term's (false)
+$shows = MyRadio_Show::getAllShows(1, !( isset($_GET["all"]) && ($_GET["all"] == 'true')));
```

Comment:
it's normally clearer to do this kind of thing into a variable which is then fed into the function call